add issue templates (#2907)
hove-io/navitia: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

> These issues are only for bugs related to the software navitia, problems on navitia.io related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia

Reviewer: ```suggestion
These issues are only for bugs related to the software navitia, problems with navitia.io or related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia
```

Reviewer 2: @nonifier I'd precise a bit more your suggestion.
```suggestion
These issues are only for bugs related to the software navitia, problems with navitia.io itself (website, access) or related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia```

**To Reproduce**

> An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.

Reviewer: ```suggestion
Please provide a navitia.io URL that reproduces the problem (you can also refer to our [Playground](http://canaltp.github.io/navitia-playground/play.html)). In the case of a custom Navitia instance, you may provide access the datasets being used.
```

Reviewer 2: @nonifier If we mention the playground, we must find a way for people to pay attention that no token is provided in URL.

> 1. An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.

Reviewer 3: ```suggestion
An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.
```

**Expected behavior**

> A clear and concise description of what you expected to happen.

Reviewer: ```suggestion
A clear and concise description of the expected behavior.
```

**Screenshots**

> If applicable, add screenshots to help explain your problem.

Reviewer: ```suggestion
If applicable, add screenshots or any resources likely to help us understanding your problem.
```
